### PR TITLE
Improve German translations for redirection control panel

### PR DESCRIPTION
**Pull request by tisto**

### CODE REVIEW

File: plone/app/locales/locales/de/LC_MESSAGES/plone.po [hunk 1]
```diff
@@ -8594,9 +8593,8 @@ msgstr "Dieser Ordner hat keinen sichtbaren Inhalt. Um Inhalte hinzuzufügen, kl
 
 #. Default: "This type has no workflow. The visibility of items of this type is determined by the folder they are in."
 #: CMFPlone/controlpanel/browser/types.py:336
-#, fuzzy
 msgid "description_no_workflow"
-msgstr "Für diesen Typ gibt es keinen Arbeitsablauf. Die Sichtbarkeit von Inhalten wird durch die Ordner bestimmt, in denen sie sich befinden."
+msgstr "Für diesen Inhaltstyp gibt es keinen Arbeitsablauf. Die Sichtbarkeit von Inhalten wird durch das Inhaltsobjekt bestimmt, in denen sich das Objekt befinden."
```
Comment (jensens) [inline plone/app/locales/locales/de/LC_MESSAGES/plone.po:8597]:
```suggestion
msgstr "Für diesen Inhaltstyp gibt es keinen Arbeitsablauf. Die Sichtbarkeit von Inhalten wird durch das Inhaltsobjekt bestimmt, in denen sich das Objekt befindet."
```

File: plone/app/locales/locales/de/LC_MESSAGES/plone.po [hunk 2]
```diff
@@ -10674,9 +10672,8 @@ msgstr "Schlagwörter werden in der Regel benutzt, um Inhalte ad hoc zu organisi
 
 #. Default: "Enter the absolute path of the target. Target must exist or be an existing alternative url path to the target."
 #: CMFPlone/controlpanel/browser/redirects-controlpanel.pt:126
-#, fuzzy
 msgid "help_target_path"
-msgstr "Geben Sie den absoluten Pfad des Ziels an. Der Pfad muss mit '/' beginnen. Das Ziel muss existieren oder eine weitere Alternative URL sein."
+msgstr "Geben Sie den absoluten Pfad des Ziels an. Der Pfad muss mit '/' beginnen. Das Ziel muss existieren oder auf eine existierende Alternative URL verweisen."
```
Comment (jensens) [inline plone/app/locales/locales/de/LC_MESSAGES/plone.po:10676]:
```suggestion
msgstr "Geben Sie den absoluten Pfad des Ziels an. Der Pfad muss mit '/' beginnen. Das Ziel muss existieren oder auf eine existierende alternative URL verweisen."
```

**Review by jensens — CHANGES_REQUESTED** [review]
Kleinigkeiten.

File: plone/app/locales/locales/de/LC_MESSAGES/plone.po [hunk 3]
```diff
@@ -10674,9 +10672,8 @@ msgstr "Schlagwörter werden in der Regel benutzt, um Inhalte ad hoc zu organisi
 
 #. Default: "Enter the absolute path of the target. Target must exist or be an existing alternative url path to the target."
 #: CMFPlone/controlpanel/browser/redirects-controlpanel.pt:126
-#, fuzzy
 msgid "help_target_path"
-msgstr "Geben Sie den absoluten Pfad des Ziels an. Der Pfad muss mit '/' beginnen. Das Ziel muss existieren oder eine weitere Alternative URL sein."
+msgstr "Geben Sie den absoluten Pfad des Ziels an. Der Pfad muss mit '/' beginnen. Das Ziel muss existieren oder auf eine existierende alternative URL verweisen."
```
Comment (davisagli) [inline plone/app/locales/locales/de/LC_MESSAGES/plone.po:10676]:
```suggestion
msgstr "Geben Sie den absoluten Pfad des Ziels an. Das Ziel muss existieren oder auf eine existierende alternative URL verweisen."
```

**Review by davisagli — APPROVED** [review]

**Pull request merged**